test/fixtures/comparison-files/lib/viewers/ResultsViewer.test.ts/one-multilocation-violation-details.txt

```diff
@@ -0,0 +1,10 @@
+Found 1 violation(s) across 1 file(s):
+=== 1. stub1RuleA
+    severity:  4 (Low)
+    engine:    stubEngine1
+    message:   This is a message
+    locations:
+        __PATH_TO_FILE_A__:20:1
+        **__PATH_TO_FILE_Z__:2:1 This is a comment at Location 2**
```

Comment:
What does ** do? Is that just to mark it as primary?

You'll want to ask Marcelino about this one I think to see if he thinks this is a good way to display things.

Reply:
Will do. Thanks.